engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/puppet-modules/content-view-puppet-modules.controller.js

```diff
@@ -26,7 +26,10 @@ angular.module('Bastion.content-views').controller('ContentViewPuppetModulesCont
         $scope.versionText = function (module) {
             var version;
             if (module['computed_version']) {
-                version = translate("Latest (Currently %s)").replace('%s', module['computed_version']);
+                if (module['uuid'])
+                    version = translate("Currently %s").replace('%s', module['computed_version']);
+                else
+                    version = translate("Latest (Currently %s)").replace('%s', module['computed_version']);
```

Comment:
Since there is only one statement in the if..else.. statement, it should not be an issue